modules/4337-gas-metering/package.json

```diff
@@ -4,7 +4,6 @@
   "description": "Gas Metering Analysis for Safe with 4337 Module",
   "homepage": "https://github.com/safe-global/safe-modules/4337-gas-metering",
   "license": "GPL-3.0",
-  "type": "module",
```

Comment:
this was affecting other project builds for some reason because they use CJS .. had to remove it

also `4337-gas-metering` had many major upgrades (@alchemy, TSX, viem), I'm unfamiliar with the codebase so I didn't push the updates